csm_web/frontend/static/frontend/data/enrollment_policies.tsx

```diff
@@ -0,0 +1,294 @@
+/*
+--SECTIONS OBJECT DOC--
+
+Format:
+  section
+  |    title
+  |    body
+  |    |  enumerated HTML content elements
+
+  anchor tag (object)
+  |   text
+  |   link
+
+Naming schema for HTML content elements:
+For a series of elements in the body, each tag type is enumerated as follows
+-Example:
+  <p> ..1.. </p>
+  <p> ..2.. </p>
+  <a> ..3.. </a>
+
+  Turns into...
+
+  body: {
+    p1: "..1..",
+    p2: "..2..",
+    a1: "..3.."
+  }
+
+In order to accommodate in-element modifications (bolding, linking, etc),
+each element with inline mods is split up into respective parts
+-Example:
+  <p> The quick brown <b>fox</b> jumps...</p>
+
+  Turns into...
+
+  body: {
+    p1_1: "The quick brown ",
+    p1_b: "fox",
+    p1_2: " jumps..."
+  }
+
+  This is then encoded into the HTML through the JS injection with React.
+*/
+
+export const SECTIONS = {
```

Comment:
Verify that this information is still consistent with the following document: https://docs.google.com/document/d/1fmzgepezzdEzfi97u_jEw3OS88gsKDh7AjpuB4pnuLA/edit#heading=h.mp1mu71frsgf

I'd recommend leaving out the specific CCN info from this page, or otherwise putting the CCN and course number into variables at the top, so they're more prominent to change for future semesters.

Reply:
I spoke with PVP about this and they said that we can leave the CCN on the page, but only if the user is enrolled in a section or should have access to it. We'll need additional validation to ensure that the viewer is a student, coordinator, or mentor before displaying it.

Reply:
i updated the page with the latest document information. i agree with making the cdn more prominent. i'm a bit confused about the getRoles function and how to incorporate it with the check because it seems like the only roles there are are student, coordinator, and mentor only.

Reply:
shall discuss on monday ig

Reply:
Yes those are the only roles, the `getRoles` function will return a `Roles` object, which you can use to see if the use is a student, coord, or mentor. Someone who logs in without enrolling in a section or being in CSM will just be a standard user and will not have any of these roles. But yeah we can discuss tomorrow to figure out details.

Reply:
decided to remove cnn and class number for simplicity